engine/execution/ingestion/engine.go

```diff
@@ -648,11 +648,12 @@ func (e *Engine) executeBlock(
 		}
 	}
 
+	finalEndState := computationResult.InterimEndState()
```

Comment:
maybe rename the method to just EndState or CurrentEndState

Reply:
renamed at [2c8e85d](https://github.com/onflow/flow-go/pull/4105/commits/2c8e85de7993695ed39ca889adcd5151170b29eb)